templates/plugin-configs/librato.conf.j2

```diff
@@ -1,4 +1,5 @@
-{% if use_librato|bool and collectd_forwarder is defined and collectd_forwarder == inventory_hostname %}
+{% if use_librato|bool %}
+{% if (collectd_forwarder is defined and collectd_forwarder == inventory_hostname) or (collectd_forwarder is not defined) %}
```

Comment:
What about setting a default value for `collectd_forwarder` to empty string? this would allow to remove the `is defined` check.